examples/waveshare-epd/epd2in66b/main.go

```diff
@@ -0,0 +1,84 @@
+package main
+
+import (
+	"image/color"
+	"machine"
+	"time"
+
+	"tinygo.org/x/drivers"
+	"tinygo.org/x/drivers/waveshare-epd/epd2in66b"
+	"tinygo.org/x/tinyfont"
+	"tinygo.org/x/tinyfont/freemono"
+)
+
+var (
+	black = color.RGBA{0, 0, 0, 0xff}
+	white = color.RGBA{0xff, 0xff, 0xff, 0xff}
+	red   = color.RGBA{0xff, 0, 0, 0xff}
+)
+
+func main() {
+	machine.Serial.Configure(machine.UARTConfig{})
+	time.Sleep(2 * time.Second)
+
+	machine.SPI1.Configure(machine.SPIConfig{
+		Frequency: epd2in66b.Baudrate,
+	})
+
+	println("started")
+
+	// in case you have a Pico module, you can directly use
+	// dev, err := epd2in66b.NewPicoModule()
```

Comment:
I've kept the example for the generic setup with this hint for the specific `pico` use case.